cmake.deps/deps.txt

```diff
@@ -59,3 +59,6 @@ TREESITTER_MARKDOWN_URL https://github.com/MDeiml/tree-sitter-markdown/archive/v
 TREESITTER_MARKDOWN_SHA256 4909d6023643f1afc3ab219585d4035b7403f3a17849782ab803c5f73c8a31d5
 TREESITTER_URL https://github.com/tree-sitter/tree-sitter/archive/v0.22.5.tar.gz
 TREESITTER_SHA256 6bc22ca7e0f81d77773462d922cf40b44bfd090d92abac75cb37dbae516c2417
+
+WASMTIME_URL https://github.com/dundargoc/wasmtime/archive/3080db76c3115c0548b9796ec13754e02e1343ba.tar.gz
```

Comment:
Just use commit on my fork for now until https://github.com/bytecodealliance/wasmtime/pull/8496 is merged or a new version is released

Reply:
supply chain

Reply:
Huh?

Reply:
@dundargoc wasmtime 20.0.1 is released with your fix now.